hack/util.sh

```diff
@@ -5,6 +5,88 @@
 TIME_SEC=1000
 TIME_MIN=$((60 * $TIME_SEC))
 
+# configure_and_start_os will create and write OS master certificates, node config,
+# OS config and finally starts the OS server. Returns the PID of the OS server.
+# 
+# $1 - log dir
+function configure_and_start_os {
+  openshift admin ca create-master-certs \
+    --overwrite=false \
+    --cert-dir="${MASTER_CONFIG_DIR}" \
```

Comment:
So I've tried to combine them and for some reason the test are failing cause the `wait_for_url`  gave up waiting for kubelet.
I wouldn't combine starting of the server and waiting for its endpoints, since there are couple of steps that we are doing between those two actions.